app/org/sagebionetworks/bridge/models/schedules/TaskScheduler.java

```diff
@@ -0,0 +1,26 @@
+package org.sagebionetworks.bridge.models.schedules;
+
+import java.util.List;
+import java.util.Map;
+
+import org.joda.time.DateTime;
+
+public interface TaskScheduler {
+
+    /**
+     * Get all the tasks for the schedule that start before up to the "until" date and time passed to the method. Tasks
```

Comment:
Does this return *all* tasks before the "until" date? Or all tasks between now and the "until" date?